dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml

```diff
@@ -319,22 +319,17 @@
     <dependency>
       <groupId>com.sun.xml.bind</groupId>
       <artifactId>jaxb-impl</artifactId>
-      <version>2.2.10</version>
-    </dependency>
-    <dependency>
-      <groupId>com.sun.xml.bind</groupId>
-      <artifactId>jaxb-impl</artifactId>
-      <version>2.3.0.1</version>
+      <version>2.3.8</version>
```

Comment:
Maybe it doesn't matter a huge amount but 2.3.8 is not the latest 2.x version, there is a 2.3.9.
Same for things like jaxws-rt, there is a 2.3.7.